app/containers/MessageBox/index.js

```diff
@@ -746,13 +854,54 @@ class MessageBox extends Component {
 					style={styles.mentionList}
 					data={mentions}
 					renderItem={({ item }) => this.renderMentionItem(item)}
-					keyExtractor={item => item._id || item.username || item}
+					keyExtractor={item => item._id || item.username || item.command || item}
 					keyboardShouldPersistTaps='always'
 				/>
 			</View>
 		);
 	};
 
+	renderCommandPreviewItem = (item) => {
+		if (item.type === 'image') {
+			return (
+				<TouchableOpacity
+					style={styles.commandPreview}
+					onPress={() => this.onPressCommandPreview(item)}
+					testID={`command-preview-item${ item.id }`}
+				>
+					<FastImage
+						style={styles.commandPreviewImage}
+						source={{
+							uri: item.value
+						}}
+						resizeMode={FastImage.resizeMode.cover}
+					/>
+				</TouchableOpacity>
+			);
+		}
+		return null;
+	}
+
+	renderCommandPreview = () => {
+		const { commandPreview } = this.state;
+		if (!this.showCommandPreview) {
+			return null;
+		}
+		return (
+			<View key='commandbox-container' testID='commandbox-container'>
+				<FlatList
+					style={styles.mentionList}
+					data={commandPreview}
+					renderItem={({ item }) => this.renderCommandPreviewItem(item)}
```

Comment:
use `renderItem={this.renderCommandPreviewItem}` if possible

Reply:
Do as Ilarion said.
renderCommandPreviewItem will accept an object instead of item.
Do it in the rest of the code.
```jsx
renderCommandPreviewItem = ({ item }) => {}
```